meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -393,7 +393,7 @@ WKS_FILE:sota:imx8mp-lpddr4-evk-ebbr ?= "efidisk-sota.wks"
 MACHINE_FEATURES:append:imx8mp-lpddr4-evk = " nxp89xx mxm-mwifiex-load"
 
 # STM32MP1
-PREFERRED_PROVIDER_virtual/kernel:stm32mp1common ?= "linux-lmp-stm32"
+PREFERRED_PROVIDER_virtual/kernel:stm32mp1common ?= "linux-lmp"
```

Comment:
We should include the meta bump commit as part of this pr as well.

Reply:
done